config-v3-documentation

```diff
@@ -55,6 +55,20 @@ data_subdir=tests/
 in_gen=src/gen
 # Source code of generator (without suffix)
 
+in_format=text
+# Format of input:
+#   text    text/plain in ASCII (default)
+#           subject to normalization (newlines, encoding)
+#   binary  application/binary
+
+out_format=text
+# Format of output:
+#   text    text/plain in ASCII (default)
+#           subject to normalization (newlines, encoding)
+#   binary  application/binary
+#
+# TODO: Server mode
```

Comment:
I'm not sure I understand the point of this TODO. Will  the format or semantics of this option check anything when the server mode is implemented?

Reply:
Improved the comment.